### ISSUE TITLE
Run gradient descent and ascent unlearning for CIFAR-10 DDPM

### ISSUE THREAD

**Issue by chris522229197** [q8888620002/Group-Attribution-for-Diffusion-Models#43]
Tune the number of steps as a hyperparameter to achieve the best data attribution performance. Aim for at least 1k unlearning instances.
